echoregions/lines/lines.py

```diff
@@ -1,29 +1,33 @@
+import json
 from typing import Dict, Iterable, List, Union
 
+import matplotlib.pyplot as plt
 from pandas import DataFrame, Series, Timestamp
 
-from ..convert.evl_parser import LineParser
-from . import Geometry
+from ..utils.io import validate_path
+from .lines_parser import parse_line_file
 
 
-class Lines(Geometry):
-    def __init__(
-        self,
-        input_file: str = None,
-        parse: bool = True,
-        nan_depth_value: float = None,
-        offset: float = 0,
-    ):
-        self._parser = LineParser(input_file)
-        self._plotter = None
-        self._masker = None
+class Lines:
+    def __init__(self, input_file: str, nan_depth_value: float = None):
+        self.depth = (
+            None  # Single array that can be used to obtain min_depth and max_depth
+        )
+        self._min_depth = (
+            None  # Set to replace -9999.99 depth values which are EVR min range
+        )
+        self._max_depth = (
+            None  # Set to replace 9999.99 depth values which are EVR max range
+        )
```

Comment:
`self.depth`, `self._min_depth`, `self._max_depth` are not used anywhere in the Line class, so I am a little unsure why they are here. Am I missing anything?